pages/docs/dev-docs/release-strategy.md

```diff
@@ -38,6 +38,19 @@ permalink: dev-docs-release-strategy.html
 5. Make sure everything is working
 6. Release R
 
+## Release schedule
+
+The core library repository follows semantic versioning.
+
+We group together breaking changes and release a new breaking version as needed, but not more frequently than once every two years (aiming for longer). This aims to give time and motivation to users to adopt new versions, and spreads the effort of preparing a release over a longer period of time. There is no fixed schedule for major releases, and we take time to ensure that no further breaking changes are needed soon after.
+
+We aim for feature releases twice per year, so that new features can be continuously shipped to new users, but we again don't get overly burdened by the release process, and feature releases maintain their significance. We define a feature freeze date (date after which no new feature contributions can be merged) and a release date (date of merging to the main branch and tagging the release). Since most of the development happens in universities, we align these dates with the lecture-free periods between semesters in these universities:
```

Comment:
I think the feature freeze is still unclear. What does this mean? We freeze `develop` between the feature freeze and the release? According to the plan below this means that up to 40 days per release and 80 days per year `develop` is blocked and no integration of new features for the complete team is possible. Do we want this?

I would prefer to create a release branch from `develop` on the date of the feature freeze since this would allow to continue working on `develop`. But we probably discussed this and there was some argument for not having a release branch.

Reply:
The "Release procedure" section in this page ([above](https://precice.org/dev-docs-release-strategy.html#release-procedure)), as well as the PR template currently specifie a release branch: https://github.com/precice/precice/blob/develop/tools/releasing/PULL_REQUEST_TEMPLATE/release_pull_request_template.md

I now modified the definition to explicitly mention the release branch.